tc-server/src/main/java/com/tc/l2/logging/TCLogbackLogging.java

```diff
@@ -51,6 +51,12 @@ public static void resetLogging() {
     root.detachAndStopAllAppenders();
     console.detachAndStopAllAppenders();
     loggerContext.reset();
+
+    try {
+      new ch.qos.logback.classic.util.ContextInitializer(loggerContext).autoConfig();
+    } catch (Exception e) {
+      ch.qos.logback.core.util.StatusPrinter.printInCaseOfErrorsOrWarnings(loggerContext);
+    }
```

Comment:
This is used for inline testing and is an attempt to shutdown everything for the current server/classloader.  The next server/classloader should do this automatically.